### DISCUSSION TITLE
Redefining the next decade of cybersecurity: AI-powered security built to empower developers - SEC2732M

### DISCUSSION THREAD

**Discussion by jricciardi** [community/community#74696] [Enterprise]
Wednesday, Nov 811:15 AM - 11:45 AM PST
AI is ushering in a new era for security, presenting some of the most promising opportunities to secure code from the start. Join us as we explore GitHub Advanced Security’s mission in this new paradigm, and how AI-powered security solutions will position developers as leaders in a global security strategy. We’ll reflect on how GitHub helped make a positive impact in application security over the last year, shed light on what’s next, and explore the intricate dynamics at play as AI becomes a cornerstone feature in the industry. Together, we'll redefine the concept of shifting security left and shape the security landscape for the next decade.

We're getting started now with @mph4! Watch along [on the Universe portal](https://reg.githubuniverse.com/flow/github/universe23/sessioncatalog/page/watchasl)

Read more about GitHub's latest Security features in our blog post **[Introducing AI-powered application security testing with GitHub Advanced Security](https://github.blog/2023-11-08-ai-powered-appsec/)**

![image](https://github.com/community/community/assets/31521880/c3d5db23-771d-4808-8b2d-3b14929af148)

Mike Hanley
Chief Security Officer and SVP of Engineering, GitHub

> Mike Hanley is the Chief Security Officer and SVP of Engineering at GitHub. Prior to GitHub, Mike was the Vice President of Security at Duo Security, where he built and led the security research, development, and operations functions. After Duo’s acquisition by Cisco for $2.35 billion in 2018, Mike led the transformation of Cisco’s cloud security framework and later served as CISO for the company. Mike also spent several years at CERT/CC as a Senior Member of the Technical Staff and security researcher focused on applied R&D programs for the US Department of Defense and the Intelligence Community. When he’s not talking about security and engineering at GitHub, Mike can be found enjoying Ann Arbor, MI with his wife and eight kids.

**Comment by jricciardi** [comment]
![Screenshot 2023-11-08 at 11 16 42 AM](https://github.com/community/community/assets/31521880/1b7c9675-8423-449e-90c7-3926c5d7a66e)

**Comment by jricciardi** [comment]
Currently on-stage: Asha Chakrabarty!
VP, Product Management, GitHub

> Asha Chakrabarty is the Vice President of Product Management for Security & Compute products at GitHub. Prior to joining GitHub, Asha was at Amazon Web Services (AWS) for 9+ years and wore multiple hats. Most recently at AWS, she led a global team of GTM and Product Specialists for AWS’ Container Services portfolio. Over the past 16+ years, her experience spans cloud computing, software development, modern application architectures, infrastructure design, and developer tools. Asha is based out of Austin, Texas where she spends her spare time with family, attempting to become a wine and dark chocolate connoisseur, and honing her novice photography skills.

![Screenshot 2023-11-08 at 11 25 09 AM](https://github.com/community/community/assets/31521880/05dee82e-e83f-438d-bfa8-ad923c29febd)

**Comment by jricciardi** [comment]
Announcing 📢  AI-generated remediation auto fixes to security issues right in your pull requests: "Review, edit, and merge fixes to vulnerabilities in real time"
![Screenshot 2023-11-08 at 11 27 11 AM](https://github.com/community/community/assets/31521880/46d0d7ee-10d3-475e-ad22-0872c553dcaf)

**Comment by jricciardi** [comment]
Read more about the Security Overview dashboard beta in this announcement: https://github.com/orgs/community/discussions/74687 

[Learn more about the new overview dashboard](https://docs.github.com/en/enterprise-cloud@latest/code-security/security-overview/viewing-security-insights-for-your-organization) and [send us your feedback](https://github.com/orgs/community/discussions/categories/code-security)

![Screenshot 2023-11-08 at 11 31 30 AM](https://github.com/community/community/assets/31521880/a5cfb880-55fb-43a2-9715-fbd57063e717)

**Comment by jricciardi** [comment]
70 improvements to GitHub's security products in 2023! That's a lot of 🚢 s! 

![Screenshot 2023-11-08 at 11 34 33 AM](https://github.com/community/community/assets/31521880/e10d0fd5-5a94-45a9-98db-4a270575fc41)

And what does that mean? 32 MILLION security issues resolved 🤯 
![Screenshot 2023-11-08 at 11 35 23 AM](https://github.com/community/community/assets/31521880/edfa5f20-9624-4d44-808f-5b6bffc9566f)

**Comment by jricciardi** [comment]
Welcome to the stage Robert (Chief Security Officer, Sierra Nevada Corporation (SNC)) and Brittany (Director PMM, GitHub)!

![Screenshot 2023-11-08 at 11 37 56 AM](https://github.com/community/community/assets/31521880/c5dbb672-7672-4597-ab0f-60a4570734ea)

Robert Daugherty
Chief Security Officer, Sierra Nevada Corporation (SNC)

> As Chief Security Officer for Sierra Nevada Corporation (SNC), Robert Daugherty leads a holistic Security Program that includes Classified Networks and Programs, Classified Facilities, Counterintelligence, Cybersecurity, Insider Risk, Personnel Security, Operational Security, Business Continuity, Crisis Management, and Safety. SNC’s Security Program is recognized by the USG as being one of the top performing in all of industry with multiple DCSA Cogswell Awards for program excellence. Armed with an MBA, BCompSc, and CISSP, CISM certifications, Robert’s extensive background includes 25+ years of moving organizations toward excellence by leveraging technology to improve mission in rapidly evolving high-growth startup business units, highly matrixed enterprises, critical infrastructure, and the Defense and Intelligence sectors. Robert’s security mantra is “assume breach”; organizations must adopt a proactive counterintelligence, data-driven and risk-based mindset, constantly hunting for and quickly detecting anomalous internal and external activity.

**Reply by jricciardi** [reply]
Awesome shot of the interview from the front row! 
![IMG_6771](https://github.com/community/community/assets/31521880/b3fdac9a-0dd5-4b91-942d-18c2162f4eed)

**Reply by jricciardi** [reply]
Loads of great quotes in this interview! But gotta love "Security is a full contact team sport"

**Comment by jricciardi** [comment]
And rounding us out with this great slide! Thanks for joining the session (or just checking out these highlights 😄 )

![IMG_6772](https://github.com/community/community/assets/31521880/ce615346-c03a-4322-8e3c-8f85fd26906a)

**Comment by Demoniker** [comment]
Well, as we embark on this journey into the next decade of cybersecurity, it's crucial to consider all our options. While GitHub's offerings are undoubtedly impressive, I believe it's wise to explore a variety of [cyber security](https://amaru.co.nz/) solutions to ensure comprehensive protection against ever-evolving threats.
In addition to GitHub Advanced Security, I'd recommend looking into other cyber security platforms and tools. Diversifying our approach to security not only strengthens our defenses but also ensures resilience in the face of diverse attack vectors.

**Comment by olivia2801** [comment]
A redefined cybersecurity landscape will likely rely on an [AI security platform](https://www.agentra.io/) that not only detects evolving threats but also integrates seamlessly into developer workflows. By learning from approaches focus on adaptive AI systems, such platforms can help developers build secure applications without adding complexity, ensuring both agility and resilience for the next decade.
